runtime.md

```diff
@@ -99,7 +97,7 @@ This operation MUST return the state of a container as specified in the [State](
 `create <container-id> <path-to-bundle>`
 
 This operation MUST [generate an error](#errors) if it is not provided a path to the bundle and the container ID to associate with the container.
-If the ID provided is not unique across all containers within the scope of the runtime, or is not valid in any other way, the implementation MUST [generate an error](#errors) and a new container MUST NOT be created.
+If the ID provided is not unique across all containers within the [scope of the container](#scope-of-a-container), or is not valid in any other way, the implementation MUST [generate an error](#errors) and a new container MUST NOT be created.
```

Comment:
> This does not make any sense to me.

Which portion?  Does the previous “scope of the runtime” make sense to you?  What does it mean to you?  Something other than the [scope of the container][1]?  Does the scope of the container (as [we currently define it][1]) make sense to you?

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc5/runtime.md#scope-of-a-container